tests/test_cases/test_force_release/Makefile

```diff
@@ -0,0 +1,30 @@
+###############################################################################
```

Comment:
This being a new file, you can use the following SPDX identifier instead of the long license text:
```
# Copyright cocotb contributors
# Licensed under the Revised BSD License, see LICENSE for details.
# SPDX-License-Identifier: BSD-3-Clause
```